src/app/app.component.html

```diff
@@ -1,6 +1,8 @@
 <nav>
   <mat-toolbar class="mat-secondary">
-    <p class="show-small"><button mat-button (click)="drawer.toggle()"><mat-icon>menu</mat-icon></button></p>
+    <p class="show-small"><button mat-button (click)="drawer.toggle()">
```

Comment:
why touch this file?

Reply:
It contains all the links to the generated features. See https://github.com/angular/angular-bazel-example/pull/358#discussion_r273439713